src/molecule/command/test.py

```diff
@@ -115,6 +116,12 @@ def execute(self):
     default=base.MOLECULE_DEFAULT_SCENARIO_NAME,
     help=f"Name of the scenario to target. ({base.MOLECULE_DEFAULT_SCENARIO_NAME})",
 )
+@click.option(
+    "--platform-name",
+    "-p",
+    default=MOLECULE_PLATFORM_NAME,
+    help=f"Name of the platform to target only. ({MOLECULE_PLATFORM_NAME} means all)",
```

Comment:
This will say "{whatever is in env} means all" which is probably not what's intended. Am I reading this correctly? Should this say something like "XXX by default"?

Reply:
Thanks for your attention. You're right.

The description is modified in new commit.